server/models/User.coffee

```diff
@@ -89,7 +90,10 @@ UserSchema.methods.isStudent = ->
 UserSchema.methods.getUserInfo = ->
   id: @get('_id')
   email: if @get('anonymous') then 'Unregistered User' else @get('email')
-  
+
+UserSchema.methods.getYearSubscriptionGroup = ->
+  core_utils.getYearSubscriptionGroup(@get('testGroupNumber'))
```

Comment:
ah, that's why. I guess that works. Feels weird to require app code from server, but I'm not sure I have a better solution other than using a server endpoint.